requirements.txt

```diff
@@ -58,7 +58,6 @@ python-bidi
 pytz
 pyyaml==6.0.1
 rauth==0.7.3
-redis==3.5.3
```

Comment:
Without the pin, in looks like redis 5.2.1 gets installed.  
Have we looked into the implications of that?  

It seems like 6.x would break, due to version conflicts with the server. 
Do we want to pin 5.x?